src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -1072,6 +1072,65 @@ class SentencingService(
       null,
     )
   }
+  fun convertToRecallSentences(offenderNo: String, request: ConvertToRecallRequest) {
```

Comment:
More or less as the same as above except each sentence now has its own sentence type.

Wil remove above function after the endpoint is deleted